vendor/bat-native-ledger/include/bat/ledger/transactions_info.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BAT_LEDGER_TRANSACTIONS_INFO_
+#define BAT_LEDGER_TRANSACTIONS_INFO_
+
+#include <string>
+#include <vector>
+
+#include "bat/ledger/export.h"
+#include "bat/ledger/transaction_info.h"
+
+namespace ledger {
+
+LEDGER_EXPORT struct TransactionsInfo {
```

Comment:
`TransactionInfoList` ? It seems way to easy to mix up TransactionInfo and TransactionsInfo

Reply:
It may not always be a list going forward